templates/tfengine/components/cicd/triggers/plan/main.tf

```diff
@@ -0,0 +1,122 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "google_cloudbuild_trigger" "plan_env" {
+  count       = var.skip ? 0 : 1
+  disabled    = var.run_on_push
+  provider    = google-beta
+  project     = var.project_id
+  name        = "tf-plan-${var.name}"
```

Comment:
tf-plan-${var.env} looks more obvious it's attaching the env name to the end

Reply:
Great idea, thanks!